.github/workflows/push-bundles.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: push-bundles
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  push-policy-bundles:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Docker login
+      uses: docker/login-action@v2
+      with:
+        # See also BUNDLE_REPO defined in Makefile which should match
+        registry: quay.io/hacbs-contract
+        username: ${{ secrets.EC_POLICY_BUNDLE_PUSH_USERNAME }}
+        password: ${{ secrets.EC_POLICY_BUNDLE_PUSH_PASSWORD }}
```

Comment:
Do the secret names feel too long? Could make them QUAY_PUSH_USER and QUAY_PUSH_PASS probably..